scripts/src/test.sh

```diff
@@ -100,3 +101,7 @@ doc_test() {
 
   __GEAR_WASM_BUILDER_NO_BUILD=1 SKIP_WASM_BUILD=1 SKIP_GEAR_RUNTIME_WASM_BUILD=1 SKIP_VARA_RUNTIME_WASM_BUILD=1 $CARGO test --doc --workspace --exclude runtime-fuzzer --exclude runtime-fuzzer-fuzz --manifest-path="$MANIFEST" --no-fail-fast "$@"
 }
+
+ignored_tests() {
+  $CARGO test -p demo-fungible-token --no-fail-fast "$@" -- --nocapture --ignored
+}
```

Comment:
mb using ft tests instead?

Reply:
I think what we could do `$CARGO test --workspace --manifest-path="$MANIFEST" --no-fail-fast "$@" -- --nocapture --ignored`, but it will probably take a long time to build.

Reply:
running all ignored tests is arbitrary, not all of the ignored tests are caused by taking too much building time

Reply:
I renamed it "time-consuming tests". In the future, we may have such tests not only in fungible-token.